app/src/main/java/org/jak_linux/dns66/db/RuleDatabaseUpdateTask.java

```diff
@@ -63,12 +65,11 @@ public RuleDatabaseUpdateTask(Context context, Configuration configuration, bool
     private void setupNotificationBuilder() {
         notificationManager =
                 (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
-        notificationBuilder = new NotificationCompat.Builder(context, NotificationChannels.UPDATE_STATUS);
-        notificationBuilder.setContentTitle(context.getString(R.string.updating_hostfiles))
-                .setContentText(context.getString(R.string.updating_hostfiles))
-                .setSmallIcon(R.drawable.ic_refresh);
-
-        notificationBuilder.setProgress(configuration.hosts.items.size(), 0, false);
+        notificationBuilder = new NotificationCompat.Builder(context, NotificationChannels.UPDATE_STATUS)
```

Comment:
`Builder`s allow chaining calls, so do that.
Remove `setContentText` as it's the same string than in the title.